src/index.css

```diff
@@ -8,4 +8,34 @@
 @font-face {
     font-family: "Montserrat";
     src: url('./fonts/Montserrat-Regular.ttf') format('truetype');
+}
+
+body {
+    margin: 0;
+}
+
+/* HEADER */
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 80px;
+    margin-right: 80px;
+}
+
+.header-logo {
```

Comment:
I'd strongly suggest not to use global CSS classes because you'll run into naming conflicts at some point or you have to use naming conventions like BEM. You can use styled-components as in the example code or alternatively CSS modules. If you choose the latter the class names are automatically scoped to the component so you won't run into conflicts even without using complex naming conventions. You can find more information about using it with create-react-app [here](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/)